godtools/App/Share/Data/MobileContentAuthTokenRepository/MobileContentAuthTokenRepository.swift

```diff
@@ -36,7 +37,14 @@ class MobileContentAuthTokenRepository {
     }
     
     func getCachedAuthToken() -> String? {
-                
-        return cache.getAuthToken()
+        
+        guard let userId = getUserId() else { return nil }
+        
+        return cache.getAuthToken(for: userId)
+    }
+    
+    func getUserId() -> String? {
```

Comment:
Same here, I would suggest returning a single type that encapsulates the authToken and userId since those are paired together.

Reply:
The issue is that they aren't actually stored together in the keychain, so it seems unnecessary
 to pull out both the token and the ID from the keychain when sometimes only the ID is needed.  Thoughts?

Reply:
Hey @rachaelblue I think having additional methods to fetch a single id and single token is okay too since the repository is just a store and in this case storing an auth object.   

I do think there should be a method that encapsulates those into a single type provided by the repository since those attributes belong to one another.  From an outside view into the repository it has know knowledge of the underlying cache.  Just that an auth token or auth object can be fetched from the store.

Reply:
Ok I made a `MobileContentAuthTokenRepoModel` that includes the token and id.  Is there a naming pattern that makes more sense for model objects that come from the repository other than "repoModel"?  We're kind of creating a separate layer between data and domain now.

Reply:
Hey @rachaelblue I think something suffixed with ```DataModel``` for the repository.  It's going to have a similar shape to our api models and cache models.  I'm wondering now if our api models should be suffixed with something other than ```DataModel```, such as ```Codable``` or ```Decodable```?  
A lot of our repositories are currently returning the underlying api codable / decodable models, but after our call yesterday, it seems we want to be creating a separate DataModel for the repositories.